frontend/package.json

```diff
@@ -24,9 +24,12 @@
     "bnc-onboard": "^1.13.0",
     "core-js": "^3.6.5",
     "ethers": "^5.5.3",
-    "quasar": "^1.0.0"
+    "quasar": "^1.0.0",
+    "vue-i18n": "8.27.1",
+    "yaml-loader": "0.6.0"
```

Comment:
What's this package for? I don't see any new yaml files added so curious why we need a yaml loader

Reply:
It's for <i18n> tags inside components: https://github.com/ScopeLift/umbra-protocol/pull/331#discussion_r875072948

Reply:
Got it, interesting!